inlang/source-code/cross-sell/cross-sell-sherlock/src/recommendation/recommendation.ts

```diff
@@ -0,0 +1,60 @@
+import { normalizePath, type NodeishFilesystem } from "@lix-js/fs"
+import { joinPath } from "./utils/joinPath.js"
+import { Value } from "@sinclair/typebox/value"
+import { parse, type CommentJSONValue, stringify } from "comment-json"
```

Comment:
@felixhaeberle will this dependency work in the browser?

Reply:
There is no limitation given by the author of this libary to I suppose that it has no node dependencies. I'll read through the source code real quick.

Reply:
nope, doesn't have Node deps in source code on the first sight.